in-adapter-api/src/test/java/gohigher/auth/support/JwtProviderTest.java

```diff
@@ -0,0 +1,140 @@
+package gohigher.auth.support;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import gohigher.global.exception.GoHigherException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+
+@DisplayName("JwtProvider 클래스의")
+class JwtProviderTest {
+
+	private static final long ACCESS_TOKEN_EXPIRE_LENGTH = 100000;
+	private static final int REFRESH_TOKEN_EXPIRE_LENGTH = 300000;
+	private static final int UNIT_TO_CONVERT_MILLI_TO_SECOND = 1000;
+	private static final String SECRET = "secretKey".repeat(6);
+
+	private final JwtProvider jwtProvider = new JwtProvider(SECRET, ACCESS_TOKEN_EXPIRE_LENGTH,
+		REFRESH_TOKEN_EXPIRE_LENGTH);
+
+	@DisplayName("createToken 메서드는")
+	@Nested
+	class createToken {
+
+		@DisplayName("value가 정해지면")
+		@Nested
+		class Value {
+			Long userId = 1L;
+			Date now = new Date();
+
+			@DisplayName("엑세스 토큰이 정상적으로 만든다")
+			@Test
+			void access_token_success() {
+				// given & when
+				String accessToken = jwtProvider.createToken(userId, now, TokenType.ACCESS);
+
+				// then
+				Claims claims = Jwts.parserBuilder()
+					.setSigningKey(Keys.hmacShaKeyFor(SECRET.getBytes(StandardCharsets.UTF_8)))
+					.build()
+					.parseClaimsJws(accessToken)
+					.getBody();
```

Comment:
```suggestion
				Claims claims = extractClaims(accessToken);
```
테스트 가독성을 위해서 메서드 분리를 하는건 어떨까요?

Reply:
좋아요!